README.md

```diff
@@ -1,2 +1,34 @@
-# docs-first-discovery
-Documentation for the First Discovery Tool
+# First Discovery Tool Documentation
+
+This project contains the content needed to build and deploy the First Discovery Tool Documentation. It is based off of the Fluid `docs-template` project.
+
+# To Build
+
+1. Clone this repository.
+2. From within the project's directory, install DocPad if it isn't already installed: `sudo npm install -g docpad`
+3. Get the required node modules: `npm install`
+4. Run docpad: `docpad run`
+5. Confirm everything is working by loading `http://localhost:9778/` in a web browser.
+
+# Getting Updates from docs-template for the First Discovery Tool Documentation
+
+Periodically `docs-template` will have updates. It is suggested you keep up to date with these changes:
+
+```
+git remote add docs-template  https://github.com/fluid-project/docs-template
```

Comment:
This seems a bit nuts. Why do we not source docs-templates as a npm module?
I think that claiming that every docs site is simply a fork of "docs-template" is asking for trouble in the long run.